File: DSA/Sauhard_Semalty_590013630/Day10Question3.py
import sys


def is_anagram(first: str, second: str) -> bool:
    "Checks whether both the strings are anagram of each other"

    if len(first) != len(second):
        return False

    return sorted(first.lower()) == sorted(second.lower())


def main() -> None:
    "The entry point of the program."

    first = input("Enter first string: ")
    if not first:
        print("Empty string is not allowed.")
        sys.exit(-1)

    second = input("Enter second string: ")
    if not second:
        print("Empty string is not allowed.")
        sys.exit(-1)

    print(is_anagram(first, second))


if __name__ == "__main__":
    main()

File: DSA/Stuti_Jain_590012149/Day_10_Question_3.c
#include<stdio.h>
int i, j, len, lenj;
char s[100], s1[100];
int main() {
    printf("Enter a string: \n");
    scanf("%s", &s);

    printf("Enter the second string: \n");
    scanf("%s", &s1);

    for(i=0; s[i]; i++){
        len++;
    }

    for(j=0; s1[j]; j++){
        lenj++;
    }

    for(i=0; i<len; i++){
        for(j=0; j<lenj; j++){
            if(s[i]==s1[j]){
                break;
            }
        }
    }

    if(len==lenj){
        printf("The strings are anagram");
    }
    else{
        printf("the strings are not an anagram");
    }

    return 0;
}
